app/views/course/survey/responses/index.json.jbuilder

```diff
@@ -0,0 +1,17 @@
+responses = @responses.map { |r| [r.course_user, r] }.to_h
```

Comment:
Can we `r.course_user_id`, I think there's no need to eager load the `exp:  :course_user` since we already loaded `@course_students `.